docs/general/builtins/aggregation.rst

```diff
@@ -619,10 +619,22 @@ Example::
 
 .. CAUTION::
 
-    Due to java double precision arithmetic it is possible that any two
+    Due to Java double precision arithmetic it is possible that any two
     executions of the aggregate function on the same data produce slightly
     differing results.
 
+.. _aggregation-stddev-pop:
+
+``stddev_pop(column)``
+----------------------
+
+The ``stddev_pop`` aggregation function is equivalent to the ``stddev``
```

Comment:
I'd suggest to do it the other way around, so move the detailed explanations from `stddev` here, and move the alias info and link from `stddev` to this one (`stddev_pop`), this way we are prepared to add the new function and change the alias from stddev to point to the new one.